examples_tests/31.SkinningDataBenchmark/common.glsl

```diff
@@ -0,0 +1,3 @@
+#define MAT_MAX_CNT 64 

```

Comment:
for this benchmark to be sane, you need far more than 64 disks, I'm thinking upwards of 16 thousand.